docs/components/modeler/web-modeler/process-application-versioning.md

```diff
@@ -1,45 +1,41 @@
 ---
 id: process-application-versioning
 title: Process application versioning
-description: Process applications allow you to deploy multiple related files together in a single bundle.
+description: Process applications allow you to create distinct versions for the entire application.
 ---
 
 :::note
 With 8.7, "milestone" has been renamed to "version". To learn more about this change, see [the related announcement](/reference/announcements/870.md#web-modeler-milestones-renamed-to-versions).
 :::
 
-Although you cannot version a process application itself, you can use [bulk version creation](versions.md#bulk-version-creation) and version tags to save a single snapshot of all the process application files in one action, instead of having to create separate versions for every file.
+Process applications support versioning, allowing you to create distinct versions for the entire application. You can use versioning to save a single snapshot of all the process application files in one action.
 
-- When you create a version in any process application file, a version is also created with the same name (version tag) for all files currently available in the process application.
+- When you create a process application version, the input version tag is applied to the `versionTag` field in the main process XML.
+- This helps you track a process application throughout its development lifecycle and ensures that the correct version is referenced.
 
-- This allows you to track a process application through the development lifecycle and ensure the correct version is called.
+## Version creation
 
-## A worked example
+To create a process application version:
 
-In this example, you have a process application that contains multiple files as well as the main process.
+1. Open any file in a process application, click on the **Create version** button in the **Versions** dropdown menu.
```

Comment:
Here's some weird UX: if I create a process app version from a form and apply a version tag, the version tag is added to a different file besides the one I'm working on.

I don't know enough about user behavior to feel comfortable adding the same tag to all resources. Instead, we can update the copy (cc @lmbateman)
(S) When I update a version tag as part of creating a new process app version, I want to know which file the version tag applies to so I know what changes
![image](https://github.com/user-attachments/assets/8d72927e-bc41-412b-b7b1-20bc97b013c9)
![image](https://github.com/user-attachments/assets/59e5a86d-8e11-4182-a51d-351b39c75ac0)

Reply:
@jonathanlukas this also relates to the question Nithin posed you

Reply:
@jonathanlukas The question was asked [here](https://github.com/camunda/web-modeler/issues/9950#issuecomment-2727575087). Your response would help us very much in making a decision here.

Reply:
@nithinssabu I thought about this a bit more and I have mixed feelings
* Having all assets with the same version tag allows better visibility in the engine
BUT
* Introducing a "revision", as Nico suggests, means we don't overload the version tag
AND 
* If a user wants to git sync a folder of dependent files (e.g. DMN), the only way is through a 'fake' PA
BUT
* If we introduce folder-level git sync, this workaround is no longer necessary

Unless there's clear user need for synced version tags, I recommend leaving the behavior as-is

Reply:
@HanselIdes Thank you 👍 I will proceed with the existing behaviour then. 

About the improvement in UX you mention here, I will wait for the input from @lmbateman and once I have it and make the changes, I will update the documentation as well.